pepper-apis/config/application.conf.ctmpl

```diff
@@ -32,6 +32,11 @@
         "dsmClientId": "{{$auth0.dsmClientId}}",
         "dsmClientSecret": "{{$auth0.dsmClientSecret}}"
     },
+    "sendGridEvents" : {
+	{{if $conf.Data.sendGridEvents.verificationKey}}
+		"verificationKey" : "{{$conf.Data.sendGridEvents.verificationKey}}"
```

Comment:
We have a `sendgrid` config object below. Will it makes sense to group this new property in there? E.g. as a new `sendgrid.eventsVerificationKey` property.

Reply:
I see in config that there are several params which also not belong to "sendgrid" section:
    "sendgridToken": "sendgridToken",
    "sendgridTemplates": {
But in general I agree that it makes sense.
But seems it needs to do the same with auth0LogEvents.token - to move it under auth0 section?

Reply:
Moved the parameter into 'sendgrid' section and renamed as suggested.